client/my-sites/plugins/plugin-management-v2/utils/get-sites-with-secondary-sites.ts

```diff
@@ -0,0 +1,16 @@
+import { DefaultRootState } from 'react-redux';
+import getNetworkSites from 'calypso/state/selectors/get-network-sites';
+import isConnectedSecondaryNetworkSite from 'calypso/state/selectors/is-connected-secondary-network-site';
+import type { SiteData } from 'calypso/state/ui/selectors/site-data';
+
+export function getSitesWithSecondarySites(
```

Comment:
Can we add a docstring for this function? It took me a while to understand what it does.

Reply:
I have added a description, thanks.